### DISCUSSION TITLE
Inquiry Regarding Sending Notifications email Using only Sunbird Backend

### DISCUSSION THREAD

**Discussion by ManojNathIC** [Sunbird-Lern/Community#196] [Query] [answered]
Hello @AmiableAnil, @rhwarrier, @amit-tarento,

As per our requirements, we need some **custom development** depending upon **sunbirdED backend** (Not frontend) that involves sending **emails** to users (notifications via emails). for that, we are trying to check **users API**, but in the user API we can see email ID's in the **masked format**.

could anyone provide guidance on how to effectively send emails to users.
cc: @komalm @manishakapadnis @paritshivani

**Comment by BharathwajShankar** [comment, marked as answer]
@ManojNathIC If you have to send emails to users, you can send the same using the below API, but before that, you have to do the below configuration setup,
https://lern.sunbird.org/use/developer-guide/user-and-org-service/configuration/email-notification-configuration
curl --location '{{host}}/api/user/v1/notification/email' \
--header 'Accept: application/json' \
--header 'Content-Type: application/json' \
--header 'Authorization: {{kong_api_key}}' \
--data-raw '{
    "request": {
        "mode": "email",
        "body": "",
        "fromEmail": "",
        "emailTemplateType": "",
        "subject": "",
        "recipientUserIds": [
            "2b9651e9-c57b-44d0-8da0-23f03ded296f"
        ]
    }
}'

please try this and let us know if you face any difficulties.

**Reply by ManojNathIC** [reply]
@BharathwajShankar  I've tested it and everything seems to be working fine. I'll let you know if I encounter any difficulties in the future. Thanks again for your help!
